index.html

```diff
@@ -0,0 +1,288 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="styles/header.css">
+    <link rel="stylesheet" href="styles/main.css">
+    <link rel="stylesheet" href="styles/footer.css">
+    <link rel="stylesheet" href="styles/media.css">
+    <link rel="stylesheet" href="fonts/fonts.css">
+    
+    <title>Document</title>
+</head>
+
+<body>
+    <header>
+        <menu>
+            <div class="menu__container">
+                <div class="menu__logo">
+                    Sitename
+                </div>
+                <nav class="menu__rightside">
+                    <div class="menu__pop-up">≡</div>
+                    <nav class="menu__pop-up-open">
+                        <ul>
+                            <li>Home</li>
+                            <li>About</li>
+                            <li>Services</li>
+                            <li>Folio</li>
+                            <li>Contact</li>
+                            <li class="funcFromSlider">loareet ipon</li>
+                        </ul>
+                    </nav>
+                    <ul class="menu__rightside_MainMenu">
+                        <li>Home</li>
+                        <li>About</li>
+                        <li>Services</li>
+                        <li>Folio</li>
+                        <li>Contact</li>
+                    </ul>
+                </nav>
+            </div>
+        </menu>
+        <div class="header__slider">
+            <img src="images/arrow-left.png" alt="" class="header__slider_leftArrow">
+            <img src="images/slider_header_1.jpg" alt="" class="header__slider-image ">
+            <div class="header__slider-container">
+                <img src="images/perosn.png" alt="" class="header__slider_image-person header__slider-container-item">
+                <div class="header__slider-mainText header__slider-container-item">Nunc digno lorem dol</div>
+                <div class="header__slider-sideText header__slider-container-item">Vivamus vestibulum nulla</div>
+                <button class="header__slider-button header__slider-container-item">Loareet ipon</button>
+            </div>
+            <img src="images/arrow-right.png" alt="" class="header__slider_rightArrow">
+        </div>
+    </header>
+    <main>
+        <div class="features">
+            <div class="features_item features_item-1">
```

Comment:
I don't like the usage of `features_item-1` like classes. If these elements are unique, you should use ids.